tests/modules/services/window-managers/i3/i3-followmouse-expected.conf

```diff
@@ -1,4 +1,4 @@
-font pango:monospace 8
+font pango:monospace 8.000000
```

Comment:
I'd say that looks a bit weird, is there a way to specify float precision when doing string interpolation with them? Although if i3 and sway parse these correctly, it's probably fine :)

Reply:
Both i3 and sway parse these correctly. I think this is the best approach because it allows for fractional font size specification.